official/recommendation/neumf_model.py

```diff
@@ -109,7 +109,11 @@ def neumf_model_fn(features, labels, mode, params):
     mlperf_helper.ncf_print(key=mlperf_helper.TAGS.OPT_HP_ADAM_EPSILON,
                             value=params["epsilon"])
 
-    optimizer = ncf_common.get_optimizer(params)
+    optimizer = tf.train.AdamOptimizer(
```

Comment:
`tf.train.AdamOptimizer` is not a valid 2.0 symbol, right? Does this code run in 2.0 like this? 
https://www.tensorflow.org/versions/r2.0/api_docs/python/tf/train

Reply:
This is in neumf_model_fn, which is only supposed to be used in the estimator case.

Reply:
Does that mean you're saying the estimator code path will not be run in 2.0? I think we should just change this to tf.compat.v1 at least, so that it will run in both 1.x and 2.0, even with the estimator codepath.

Reply:
Done.